### ISSUE TITLE
Fix focus issues with Visual Studio commands

### ISSUE THREAD

**Issue by jaredpar** [VsVim/VsVim#1377]
This issue is migrated from the following stack overflow question 

> http://stackoverflow.com/questions/24022887/can-vsvim-map-keys-to-resharper-resharper-extendselection

The root cause problem here is that when VsVim executes a Visual Studio command it does so with the keyboard focus set to the command margin.  Extensions like ReSharper use the keyboard focus to grab the context on which to execute a command.  They know nothing about the command margin and hence fail to execute.  The focus needs to be inside the text editor for this to work
